Doc/library/curses.rst

```diff
@@ -1294,6 +1296,9 @@ Several constants are available to specify character cell attributes:
 | ``A_UNDERLINE``  | Underline mode.               |
 +------------------+-------------------------------+
 
+.. versionadded:: 3.7
+    ``A_ITALIC`` was added.
+
```

Comment:
@berkerpeksag if you don't know what other options you should first read them. I have said in my comment the documentation problem needs another issue, not in this one.

Adding the note as you said is not suitable. Why do you only document A_ITALIC not other options? e.g. A_LEFT, A_LOW. A_ITALIC should be solved with them but not singly added which IMHO leads to confusion.